doc/puppetfile.mkd

```diff
@@ -109,11 +109,20 @@ latest version available.
 
     mod 'puppetlabs/apache', :latest
 
+An explicit type and/or version can be specified using the standard interface,
+`:type` and `:version`. 
+
+    mod 'puppetlabs/apache',
+      type:    'forge',
+      version: '6.0.0'
+
+Note that at present the `:source` parameter is permitted, but not implemented.
```

Comment:
I think based on that, I'm going to remove it, and just say that a `:source` parameter isn't supported for individual forge modules. At present, r10k will only let you specify a Forge source once, globally. We could add individual forge sources, but there isn't a use case reason to right now.

Reply:
ah right, I forgot to circle back to this after reading the rest of the PR and understood things better.
yeah, I think it would be clearer to say something like
> Note: The `:source` parameter can be specified for Forge modules, but is currently ignored.